### PR TITLE
remove header and change download file extension to .tsv

### PR DESCRIPTION
**Pull request by irina060981**
For the issue https://github.com/alpheios-project/alpheios-core/issues/406

- removed headers
- change extension file to `.tsv`

For the issue https://github.com/alpheios-project/alpheios-core/issues/408

- fixed punctuation list for syriac model

For the issue https://github.com/alpheios-project/alpheios-core/issues/404

- add lang attribute to wrditem component to use a correct font

### CODE REVIEW

File: packages/data-models/src/syriac_language_model.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ for the current node
    * @returns {string} a string containing valid puncutation symbols
    */
   static getPunctuation () {
-    return "\u0700\u0701\u0702\u0703\u0704\u0705\u0706\u0707\u0708\u0709\u070A\u070B\u070C\u070D\u070F-.,;:!?'\"(){}\\[\\]<>/\\\u00A0\u2010\u2011\u2012\u2013\u2014\u2015\u2018\u2019\u201C\u201D\n\r\u200C\u200D"
+    return "\u0700\u0701\u0702\u0703\u0704\u0705\u0706\u0707\u0708\u0709\u070A\u070B\u070C\u070D\u070F\\-\\.,;:!?'\"(){}\\[\\]<>/\\\u00A0\u2010\u2011\u2012\u2013\u2014\u2015\u2018\u2019\u201C\u201D\n\r\u200C\u200D"
```
Comment (balmas) [inline packages/data-models/src/syriac_language_model.js:93]: Can you apply the same fix this to all of the language model classes? Thank you.

File: packages/components/src/vue/components/word-list/download-confirmation.vue [hunk 2]
```diff
@@ -175,8 +175,8 @@
           dataForDownload = this.prepareDownloadListFull()
         }
 
-        const result = Download.collectionToCSV(dataForDownload.delimiter, dataForDownload.exportFields)(dataForDownload.wordlistData)
-        Download.downloadBlob(result, `wordlist-${this.languageCode}.csv`)
+        const result = Download.collectionToCSV(dataForDownload.delimiter, dataForDownload.exportFields, false)(dataForDownload.wordlistData)
+        Download.downloadBlob(result, `wordlist-${this.languageCode}.tsv`)
```
Comment (balmas) [inline packages/components/src/vue/components/word-list/download-confirmation.vue:179]: Hmm. This changes it for both downloads, right? The tsv extension only really applies for the flashcard download though.  It should stay csv for the other.

File: packages/data-models/src/syriac_language_model.js (hunk 1 above)
Comment (irina060981) [reply packages/data-models/src/syriac_language_model.js:93]: ok - done

File: packages/components/src/vue/components/word-list/download-confirmation.vue (hunk 2 above)
Comment (irina060981) [reply packages/components/src/vue/components/word-list/download-confirmation.vue]: you are quite right - I missed it! thank you for the catch
fixed

**Comment by irina060981** [comment]
@balmas, as fixes are not big - I will merge it after I fixed them

**Pull request merged**